Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/usr/local/lib/python3.10/dist-packages/nbclient/client.py", line 1314, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/usr/local/lib/python3.10/dist-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
  File "/usr/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/usr/local/lib/python3.10/dist-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/usr/local/lib/python3.10/dist-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/usr/local/lib/python3.10/dist-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import numpy as np
A = np.array ([[1,2,3],[3,4,5],[4,5,6]])
B = np.array ([[2,3,4],[4,5,6]])
hasil = np.dot (A,B)
print(hasil)
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mValueError[0m                                Traceback (most recent call last)
[0;32m<ipython-input-1-9c69059c4c3b>[0m in [0;36m<cell line: 4>[0;34m()[0m
[1;32m      2[0m [0mA[0m [0;34m=[0m [0mnp[0m[0;34m.[0m[0marray[0m [0;34m([0m[0;34m[[0m[0;34m[[0m[0;36m1[0m[0;34m,[0m[0;36m2[0m[0;34m,[0m[0;36m3[0m[0;34m][0m[0;34m,[0m[0;34m[[0m[0;36m3[0m[0;34m,[0m[0;36m4[0m[0;34m,[0m[0;36m5[0m[0;34m][0m[0;34m,[0m[0;34m[[0m[0;36m4[0m[0;34m,[0m[0;36m5[0m[0;34m,[0m[0;36m6[0m[0;34m][0m[0;34m][0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m      3[0m [0mB[0m [0;34m=[0m [0mnp[0m[0;34m.[0m[0marray[0m [0;34m([0m[0;34m[[0m[0;34m[[0m[0;36m2[0m[0;34m,[0m[0;36m3[0m[0;34m,[0m[0;36m4[0m[0;34m][0m[0;34m,[0m[0;34m[[0m[0;36m4[0m[0;34m,[0m[0;36m5[0m[0;34m,[0m[0;36m6[0m[0;34m][0m[0;34m][0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 4[0;31m [0mhasil[0m [0;34m=[0m [0mnp[0m[0;34m.[0m[0mdot[0m [0;34m([0m[0mA[0m[0;34m,[0m[0mB[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      5[0m [0mprint[0m[0;34m([0m[0mhasil[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;31mValueError[0m: shapes (3,3) and (2,3) not aligned: 3 (dim 1) != 2 (dim 0)

